### DISCUSSION TITLE
Can sidecar be used to run dynamic Node.js code on lambda  ?

### DISCUSSION THREAD

**Discussion by spaceworkplatform** [hammerstonedev/sidecar#112] [Q&A]
Hey,

Im trying to figure out a way to give a user a code editor in my platform to insert node code (including imports packages) and then run this user defined code with Sidecar - actual deploy the custom code function (node.js based) run it and then delete it

Think about all the cloud editor platforms or automations platforms that give you run any node.js code as part of a workflow for example, that use case exactly.

There is any way (even with tweaks needed) to make that happen with Sidecar ?

Thanks

**Comment by spaceworkplatform** [comment]
Anyone ?

**Comment by wilsenhc** [comment]
I guess you can do that.

Build a custom Lambda function that receives some code (in text form) and some arguments for said code and executes it inside the lambda function

**Comment by aarondfrancis** [comment]
I think you could, you'd have to send the code over the wire and `eval` it on the other side. That's a bit outside the scope of the help I'm able to offer though!
